### PR TITLE
[1LP][RFR] Fixed test_widgets_on_dashboard

### PR DESCRIPTION
**Pull request by quarckster**
{{pytest: -v -k test_widgets_on_dashboard}}

### CODE REVIEW

File: cfme/intelligence/reports/dashboards.py [hunk 1]
```diff
@@ -243,18 +245,16 @@ def update(self, updates):
...
+            updates["widget_picker"] = updates.pop("widgets")
         changed = view.fill(updates)
         if changed:
             view.save_button.click()
         else:
             view.cancel_button.click()
         view = self.create_view(DefaultDashboardDetailsView)
         assert view.is_displayed
-        if changed:
-            view.flash.assert_success_message('Dashboard "{}" was saved'.format(self.name))
```
Comment (rlbabyuk) [inline cfme/intelligence/reports/dashboards.py]: **Required**: Better to move it to one of the tests, or add  a TODO to be moved to tests in scope with another flash messages

**Comment by quarckster** [comment]
PRT fail is not related to the changes of the PR.

File: cfme/intelligence/reports/dashboards.py [hunk 2]
```diff
@@ -243,18 +245,22 @@ def update(self, updates):
...
```
Comment (rlbabyuk) [inline cfme/intelligence/reports/dashboards.py]: **Required**: Better to move it to one of the tests, or add a TODO to be moved to tests in scope with another flash messages

Comment (quarckster) [reply cfme/intelligence/reports/dashboards.py]: Done

**Pull request merged**